appgate/files.py

```diff
@@ -215,6 +247,15 @@ def name_or_id(values: dict[str, Any]) -> str:
                 ),
                 field=self.name,
                 load_external=should_load_file(self.operator_mode),
-            )
+            ),
+            CustomEntityLoader(
```

Comment:
mmmm can not the new field be added into the appgate metadata directly? is really needed a new custom loader? Think that if we have it in the appgate metadata from the beginning we could simplify the FileAttribLoader